diplom.py

```diff
@@ -40,44 +57,26 @@ def parse(line):
     multi_class = 'multinomial', max_iter = 100)
 print('LogisticRegression score: %f'
       % logistic.fit(train_X, train_y).score(test_X, test_y))
-Predicted_probability_distribution = logistic.predict_proba(test_X)
-#print logistic.classes_
+predicted_probability_distribution_test = logistic.predict_proba(test_X)
+predicted_probability_distribution_train = logistic.predict_proba(train_X)
+list_of_classes = logistic.classes_
 
-AP = 0
-k = -1
-for row in Predicted_probability_distribution:
-    k += 1
-    r =[]
-    for i in xrange(0, len(logistic.classes_)):
-        r.append([row[i], logistic.classes_[i]])
-    r.sort(reverse=True)    
-    for j in range (0, 5):
-        if r[j][1] == test_y[k]:
-            AP += 1.0/(j+1)
-MAP = AP / TEST_DATASET_SIZE
-print (MAP)
+MAP_test = mean_average_precision(predicted_probability_distribution_test, 
+    list_of_classes, 
```

Comment:
You can use same practice of aligning arguments here - it improves code readability.